pkg/plugin/v3/scaffolds/api.go

```diff
@@ -85,23 +86,26 @@ func (s *apiScaffolder) scaffold() error {
 	if s.doResource {
 		s.config.AddResource(s.resource.GVK())
 
```

Comment:
all the changes made in this file are not required at all by following by suggestion made in https://github.com/kubernetes-sigs/kubebuilder/pull/1644/files#r510025737 and can be reverted.